toolchain/parse/testdata/function/definition/builtin.carbon

```diff
@@ -8,13 +8,35 @@
 // TIP: To dump output, run:
 // TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/parse/testdata/function/definition/builtin.carbon
 
```

Comment:
Why not merge these into `definition.carbon`?

Reply:
Merged.